templates/kommander.yaml

```diff
@@ -29,8 +29,8 @@ spec:
       enabled: true
   chartReference:
     chart: kommander
-    repo: https://mesosphere.github.io/charts/stable
-    version: 0.2.13
+    repo: https://hectorj2f.github.io/charts-1/stable
```

Comment:
I am running a second check with Konvoy CI, but this worked for me when I tested it for the http proxy. I will change it back to mesosphere when everything is ✅